tools/Resend.ApiServer/Controllers/EmailController.cs

```diff
@@ -62,19 +62,26 @@ public EmailReceipt EmailRetrieve( [FromRoute] Guid id )
     /// <summary />
     [HttpPost]
     [Route( "emails/batch" )]
-    public ListOf<ObjectId> EmailBatch(
+    public EmailBatchResponse EmailBatch(
         [FromHeader( Name = "Idempotency-Key" )] string? idempotencyKey,
+        [FromHeader( Name = "x-batch-validation" )] string? validationMode,
         [FromBody] List<EmailMessage> messages )
     {
         _logger.LogDebug( "EmailBatch" );
 
         if ( idempotencyKey != null )
             _logger.LogDebug( "With {IdempotencyKey}", idempotencyKey );
 
-        var list = new ListOf<ObjectId>();
-        list.Data = messages.Select( x => new ObjectId()
+        if ( validationMode != null )
+            _logger.LogDebug( "With {ValidationMode}", validationMode );
+
+
+        /*
+         *
+         */
+        var list = new EmailBatchResponse();
+        list.Data = messages.Select( x => new EmailBatchReceipt()
         {
-            Object = "email",
```

Comment:
I think it's fine to remove this as it makes the response compatible with the API, but just highlighting this takes a tiny chance of breaking consumers who rely on `object` (extremely unlikely though).

I understand that the API does not return `object` (I checked the `git blame` for the API and docs as well) so I think we should go in this direction, but just thought I'd bring this up anyway.